### DISCUSSION TITLE
Problem Downloading Giada 0.18.0.1 for macOS (64 bit)

### DISCUSSION THREAD

**Discussion by cpsmusic** [monocasual/giada#477] [General]
Hi All,

I'd like to download Giada 0.18.0.1 for macOS (64 bit) however I keep getting an error message:

"giada 2 is damaged and can't be opened".

I'm on macOS 10.15.7 using Firefox and Keka for the extraction.

Cheers,

Chris

**Comment by gvnnz** [comment]
Hi @cpsmusic , we are currently investigating the problem in #475 . As I suggested in the issue, could you try to you unpack the executable from the .app package and run it directly?

**Reply by cpsmusic** [reply]
I tried running the .app directly however I'm getting the same error message.

**Comment by gvnnz** [comment]
OK this is due to the quarantine flag attached to executables downloaded from the Internet. As a temporary workaround, try run the following command in the terminal:

```
sudo xattr -rd com.apple.quarantine [path/to/giada.app]
```

Sources:

https://eclecticlight.co/2020/10/29/quarantine-and-the-quarantine-flag/
https://craftware.xyz/tips/Quarantine-file-attribute.html
